.github/workflows/docker_image_main_branch_arm.yml

```diff
@@ -6,10 +6,16 @@ on:
       - docker/**
 jobs:
   build-and-push:
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
     steps:
       - name: deploy
         uses: actions/checkout@v2
+      - name: Set up QEMU
```

Comment:
I don't think buildx is building an actual arm image.  Since the `build-and-push` action runs on ubuntu (amd64), just having `QEMU` ready isn't enough, buildx has to know to build for that platform and work with `QEMU`.  What is actually happening here is `buildx` is building for the native platform (amd64) and adding the `arm` platfrom tag.